tests/e2e/vLLM/run_tests.sh

```diff
@@ -2,10 +2,23 @@
 
 SUCCESS=0
 
-# Parse list of configs.
-MODEL_CONFIGS="$PWD/tests/e2e/vLLM/configs"
+while getopts "c:t:" OPT; do
+  case ${OPT} in
+    c ) 
+        CONFIG="$OPTARG"
+        ;;
+    t )
+        TEST="$OPTARG"
+        ;;
+    \? )
+        usage
```

Comment:
The `usage` function is not defined anywhere, so this attempt to call it will result in an error that will mask the real error.

Reply:
Same as what Domenic mentioned above. I'll update the test workflow to include the lm eval tests once this PR is landed.

Reply:
Will update

Reply:
The code seems not updated yet to include the usage function definition?